### PR TITLE
Add clarification of expectations for Introduction of README

### PR DESCRIPTION
**Pull request by jmaddox19**

### CODE REVIEW

**Review by tildeee — APPROVED** [review]
Yes! Correct!

I'm a little unsure why your tests look like that though... but otherwise the clarifications look good

**Comment by jmaddox19** [comment]
Whatever we need to do to make `test_helper` initialize with the minitest require statements needs to be done

**Comment by jmaddox19** [comment]
I have a couple propositions for reduction of scope without losing any key learning objectives:
1. Do away with the Trips page. It seems odd to me to have pages for trips when it is only going to show information you can see from the Rider and Driver pages anyway. It is also a good example of a case where it might make sense NOT to support all of the "7 RESTful Routes". This will reduce a lot of work because it mean the students only need to support the creation of Trips and the rating of trips. No further editing or deleting of them.
2. Do away with the requirements that involve deleting anything, including Drivers and Passengers. This does away with some complications regarding displaying trips for Drivers/Passengers that have been deleted and it seems to me like deleting your account so that the Driver/Passenger you rode with don't have record of who you were would be real sketchy and not actually supported by a real application anyway. 
The con of #2 is if we feel they haven't had enough practice with the delete action specifically.

File: test/controllers/trips_controller_test.rb [hunk 1]
```diff
@@ -1,23 +1,7 @@
 require "test_helper"
 
 describe TripsController do
-  describe "show" do
```
Comment (tildeee) [inline test/controllers/trips_controller_test.rb]: Again, I'm unsure about why these test stubs got deleted?

**Review by tildeee — CHANGES_REQUESTED** [review]
To reply to @jmaddox19 's comment from above:

Hm, I like #1 better... my biggest concern is that without a Trips edit page, then I feel that "rating" a trip will definitely require its own custom route... There's been a history where students accomplish Trip ratings either through a custom route, or by utilizing the edit page, and "the edit page" is the more direct solution for students who struggle with Rails... Therefore, if we make this change (to cut "edit Trips"), then I'd like there to be just a new wireframe for the Trips detail page that reflects how to change the rating of the Trip without going to the edit page, and detail on the project README to help support that clearly


As for #2, in my opinion, the messiness of how Rails deals with deletes... is actually "relatively straightforward" to carry through with some Google research, and what students get out of it is the work of anticipating those effects. Granted, that's not a formal/major learning goal, but it's one that I like. If we take out this feature, I'd like to turn it into an optional enhancement.


Out of curiosity, was trying to find options for how to reduce project scope something that works towards the completion of the Trello cards "Better test scaffolding in RSR" or "RideShareRails needs some TLC"? Just curious about where the suggestions for reducing project scope came from.

**Pull request merged**